build/build.ps1

```diff
@@ -22,14 +22,14 @@ function Build-One {
 
     Write-Host "##[info]Building $project ..."
     if ("" -ne "$Env:ASSEMBLY_CONSTANTS") {
-        $args = @("/property:DefineConstants=$Env:ASSEMBLY_CONSTANTS");
+        $extra_args = @("/property:DefineConstants=$Env:ASSEMBLY_CONSTANTS");
```

Comment:
[nit, please feel free to ignore] Generally, PowerShell style for variables is to use `$camelCase`, such that I would suggest `$extraArgs` here.